packages/documentation/copy/en/javascript/JSDoc Reference.md

```diff
@@ -663,6 +663,37 @@ function box<U>(u: U): Box<U> {
   return { t: u };
 }
 ```
+sometimes, you don't want to link a type, instead, link a prop defined in a type:
```

Comment:
```suggestion

You can also link a property:
```

Reply:
@jakebailey Thank you, it looks short and clear.